samples/vsphere/common/connect_with_cert.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+"""
+* *******************************************************
+* Copyright (c) VMware, Inc. 2017. All Rights Reserved.
+* SPDX-License-Identifier: MIT
+* *******************************************************
+*
+* DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+* EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+* WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+* NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+"""
+
+__author__ = 'VMware, Inc.'
+__copyright__ = 'Copyright 2017 VMware, Inc. All rights reserved.'
+__vcenter_version__ = '6.0+'
+
+import atexit
+from com.vmware.cis.tagging_client import (Category, CategoryModel)
+from samples.vsphere.common import vapiconnect
+from samples.vsphere.common.sample_util import process_cli_args
+from samples.vsphere.common.sample_cli import build_arg_parser
+
+
+class CertConnect(object):
+    """
+    Demonstrates how to Connect to vCenter vAPI service with
+    with Valid Cert
+    """
+
+    def __init__(self):
+        self.server = None
+        self.username = None
+        self.password = None
+        self.stub_config = None
+        self.cleardata = None
+        self.skip_verification = False
+        self.cert_path = None
+        self.category_svc = None
+        self.category_id = None
+
+    def setup(self):
+        parser = build_arg_parser()
+        parser.add_argument('-cpath', '--cert_path',
+                            action='store',
+                            help='Verify vCenter Server certificate')
```

Comment:
The help message should be more precise. The requests documentation says"path to a CA_BUNDLE file or directory with certificates of trusted CAs". How about we use the same?

http://docs.python-requests.org/en/master/user/advanced/#ca-certificates